python/nano/setup.py

```diff
@@ -66,9 +66,9 @@ def download_libs(url: str):
 
 def setup_package():
 
-    tensorflow_requires = ["intel-tensorflow==2.7.0",
```

Comment:
let's make the version stays at 2.7.0 since not all features is supported in 2.9.0